### PR TITLE
auth: pam fallback log message and always call pam_end after pam_authenticate

### PR DESCRIPTION
**Pull request by PaideiaDilemma**
Changed the log, because someone used an absolute path for `general:pam_module`.

Also new release maybe?
People that use 0.3.0 are quite a bit behind by now and don't have d9f644125468054d2a237adf4ae676f64c0da918.

### CODE REVIEW

File: src/core/Auth.cpp [hunk 1]
```diff
@@ -91,6 +91,8 @@ bool CAuth::auth() {
     }
 
     ret = pam_authenticate(handle, 0);
+    pam_end(handle, ret);
+    handle = NULL;
```
Comment (vaxerski) [inline src/core/Auth.cpp:95]: nullptr

**Comment by vaxerski** [comment]
> Also new release maybe?

sure.

File: src/core/Auth.cpp (hunk 1 above)
Comment (PaideiaDilemma) [reply src/core/Auth.cpp]: thx

**Review by vaxerski — APPROVED** [review]
thanks

**Pull request merged**